wordpress-mobile/WordPress-Android
# Jetpack Focus: Emphasize uninstalling the app in the migration success card

Fixes #18262

## Description
Updates the migration success card design to make clear to users they should **uninstall the WordPress app** (not delete their site)

Design ref: qk5s4q4mZsXknwhMWo2vIY-fi-10230_21914
Matching iOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/20515

## To test

1. Delete the Jetpack app
2. Install the WordPress app if not installed already
3. Login in the WordPress app if not already
4. Build and run the Jetpack app
5. Follow the migration flow
6. *Verify* that the migration success card should match the new designs - please test light vs dark mode, landscape vs portrait, min / max font sizes
7. Tap on the migration success card
8. *Verify* that the overlay should match the new designs  - please test light vs dark mode, landscape vs portrait, min / max font sizes (note: we've decided to use ["on your device" instead of "on your iPhone"](p1681296746617889/1681290711.175429-slack-C0180B5PRJ4)) 

|Before | After (light mode) | After (dark mode)|
|---|---|---|
|![Screenshot_20230413_200404](https://user-images.githubusercontent.com/304044/231833557-cf937dd8-ece2-4ae8-85a4-0eafe55d53d1.png)|![Screenshot_20230413_200504](https://user-images.githubusercontent.com/304044/231833595-17b226a2-db76-4074-8e8d-d28c55b47bb4.png)|![Screenshot_20230413_200532](https://user-images.githubusercontent.com/304044/231833616-d029337c-341e-4df8-a233-2874b2b1f12c.png)|
|![Screenshot_20230413_200417](https://user-images.githubusercontent.com/304044/231833749-8a3c46be-d613-43f8-888c-93f996f1e965.png)|![Screenshot_20230413_200515](https://user-images.githubusercontent.com/304044/231833774-76a8659c-19f7-45cf-b6b7-32b7504c05ba.png)|![Screenshot_20230413_200541](https://user-images.githubusercontent.com/304044/231833803-1ba7840f-c2f0-4a98-b2fb-adbafbc11037.png)|

|Final migration step|After RTL (light mode)|After RTL (dark mode)|
|---|---|---|
|![Screenshot_20230413_204115](https://user-images.githubusercontent.com/304044/231840976-a7b8e8e9-67db-4e08-9461-1ba22d9d1acf.png)|![Screenshot_20230413_202342](https://user-images.githubusercontent.com/304044/231837366-2944ed7a-69f5-439e-8ddc-9b32ccc06dbd.png)|![Screenshot_20230413_202327](https://user-images.githubusercontent.com/304044/231837394-0b513859-bca7-4e12-b53f-880d3aec1ece.png)|

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
